src/fzf.rs

```diff
@@ -0,0 +1,21 @@
+use std::process::{Command, Stdio};
+use std::str;
+use which::which;
+
+pub fn is_fzf_installed() -> bool {
+    which("fzf").is_ok()
+}
+
+pub fn fuzzy_find_config() -> Option<String> {
```

Comment:
This should return a `Result<String, anyhow::Error>` instead of panicking at various points within, especially since we're shelling out and parsing OS strings

Reply:
Hopefully addressed in latest commit.